app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -314,7 +314,6 @@ type SelectedCustomizableOption {
 type SelectedCustomizableOptionValue {
     id: Int!
     label: String!
-    value: String
```

Comment:
Please mark the field as deprecated instead of removing it.

Reply:
The field was marked as deprecated.